.github/workflows/translation_status.yml

```diff
@@ -33,4 +33,4 @@ jobs:
       - name: Upload translation status CSV to Dune
         env:
           DUNE_API_KEY_PROD: ${{ secrets.DUNE_API_KEY_PROD }}
-        run: "python scripts/upload_translation_status.py"
\ No newline at end of file
+        run: "python scripts/upload_translation_status.py"
```

Comment:
the description mentioned removing this workflow altogether, do we want to do that? looks like here nothing is happening?

Reply:
Decided to keep it just to have the list available on the dashboard and just noticed there was a bug. Fixed the workflow file and edited the pr description